### ISSUE TITLE
[BUG REPORT] 522 error, incorrect voice server

### ISSUE THREAD

**Issue by piokrek** [ottomated/CrewLink#361]
1. [ ] I am not asking a question => use the [Discord](https://discord.gg/9mwuVNA) if you are
2. [yes ] I am using the Steam version of Among Us
3. [ yes] I have tried to use an [alternative voice server](https://status.crewl.ink/)
4. [yes ] I have checked everyone in my lobby is on the same CrewLink server
5. [ ] I have used `Ctrl+R` on the CrewLink app when I can't hear some people (this is a known issue)
6. [yes ] I have checked that the CrewLink server I'm using is up to date
7. [no ] I have a screenshot of any errors
8. [ yes] I have checked that someone else has not reported this using the [search bar](https://github.com/ottomated/CrewLink/issues?q=is%3Aissue)

**Describe the bug**
I started the crewlink, in voice server i paste "https://status.crewl.ink/" and also my discord invitation link, nothing work. There is always an window with 522 error: incorrect voice server, close the app and run again when you fix it. <= something like that, it's not 1:1 what i see.

**To Reproduce**
Steps to reproduce the behaviour:

**Expected behaviour**
I would like to know, what i am doing wrong, or if it's common problem, how to fix it or how long i have to wait on new version of crewlink with fixed problem

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Version: [1.1.6]

**Comment by IchigoBunBun** [comment]
![AmongUSIssue](https://user-images.githubusercontent.com/75934786/102011111-872de580-3d42-11eb-8ada-fc6aa3e4e9c3.PNG)
 Same problem here.

**Comment by SvenvtH** [comment]
As shown on https://status.crewl.ink/ you can see that the [Official Crewlink server](https://status.crewl.ink/786580536) is down.
Also, https://status.crewl.ink/ isn't the voice server, it's a website where you can check the downtime/status of some Crewlink servers. You can pick any of those on the site. 
Here a list with current crewlink servers:
[Cobchise - NA](https://status.crewl.ink/786580547) https://crewlink.among-us.tech
[Lermatroid - NA](https://status.crewl.ink/786584478) https://crewlink.glitch.me
[The Skeld 1 - NA](https://status.crewl.ink/786580537) https://s1.theskeld.xyz
[The Skeld 2 - NA](https://status.crewl.ink/786580540) https://s2.theskeld.xyz
[The Skeld 3 - NA](https://status.crewl.ink/786580543) https://s3.theskeld.xyz
[The Skeld 4 - EU](https://status.crewl.ink/786584348) https://s4.theskeld.xyz
[ubergeek77 - NA](https://status.crewl.ink/786580554) https://public2.crewl.ink

Try using one of those servers or creating your own server [here](https://github.com/ottomated/CrewLink-server).

**Comment by AMBrown21** [comment]
You can also use this voice server as well since the official server is down at the moment of me posting this comment: https://eu2-crewlink.herokuapp.com/

**Comment by MatthewDVance** [comment]
> You can also use this voice server as well since the official server is down at the moment of me posting this comment: https://eu2-crewlink.herokuapp.com/

how did you get that to work? I deployed using the heroku instructions and get the "unsupported version" error

**Comment by SvenvtH** [comment]
> > You can also use this voice server as well since the official server is down at the moment of me posting this comment: https://eu2-crewlink.herokuapp.com/
> 
> how did you get that to work? I deployed using the heroku instructions and get the "unsupported version" error

Please create another issue for this.

**Comment by AMBrown21** [comment]
All I did was put https://eu2-crewlink.herokuapp.com/ as the voice server. It worked fine, even last night when I streamed Among Us with proximity chat last night.
